packages/flutter_tools/test/commands.shard/hermetic/daemon_test.dart

```diff
@@ -121,10 +123,15 @@ void main() {
 
       expect(response.data['id'], 0);
       expect(response.data['result'], isNotEmpty);
-      expect((response.data['result']! as Map<String, Object?>)['platforms'], <String>{'macos'});
+      expect((response.data['result']! as Map<String, Object?>)['platforms'], const <String>{'macos', 'preview'});
```

Comment:
Same here - the strings here should match `platformType`s, so it the preview device has a platformType of `windows`, then I don't think we'd expect `preview` to come back here because VS Code is comparing the results of `getSupportedPlatforms` (which is badly named and should really have been `getSupportedPlatformTypes`) with the `platformType` of each device.